servicecontrol/servicecontrol-instances/configuration.md

```diff
@@ -124,6 +124,25 @@ A configuration that specifies one or more attached Audit instances. See also [S
 | --- | --- |
 | string | None |
 
+### ServiceControl/ShutdownTimeout
+
+_Added in version 6.5.0_
+
+The maximum allowed time for the process to complete the shutdown.
+
+| Context | Name |
+| --- | --- |
+| **Environment variable** | `SERVICECONTROL_SHUTDOWNTIMEOUT` |
+| **App config key** | `ServiceControl/ShutdownTimeout` |
+| **SCMU field** | N/A |
+
+| Type | Default value |
+| --- | --- |
+| TimeSpan | `00:00:05` (5 seconds) |
```

Comment:
Instead of splitting, maybe better to use a single table like:

```suggestion
| Containers | TimeSpan | `00:00:05` (5 seconds) |
| Powershell | TimeSpan | `00:02:00` (2 minutes) |
| ServiceControl Management Utility (SCMU) | TimeSpan | `00:02:00` (2 minutes) |
```

Even though its mentioned in the NOTE I didn't read that at first